src/librustdoc/formats/item_type.rs

```diff
@@ -52,7 +52,7 @@ pub(crate) enum ItemType {
     AssocConst = 19,
     Union = 20,
     ForeignType = 21,
-    OpaqueTy = 22,
+    // OpaqueTy used to be here, but it was removed in #127276
```

Comment:
CC @notriddle, as this is part of the rustdoc-the-binary<->rustdoc-search boundry